virttest/graphical_console.py

```diff
@@ -0,0 +1,242 @@
+"""
+Graphical Console
+"""
+
+import time
+import os
+import csv
+
+from virttest import data_dir
+
+
+_STR_DASH = '-'
+
+KEYMAP_DIR = os.path.join(data_dir.get_shared_dir(), 'keymaps')
+
+
+def get_keymap(keymap_file):
+    """
+    Get Key Mapping Table.
+
+    :param keymap_file: a csv file that includes key mapping table info.
+                        eg: "keymap_to_qcode.csv"
+    :return: key mapping table info as a dict.
+    """
+    keymap = os.path.join(KEYMAP_DIR, keymap_file)
+    with open(keymap, "r") as f:
+        reader = csv.reader(f)
+        first = True
+        keymap_dict = {}
+        for row in reader:
+            # Discard column headings
+            if first:
+                first = False
+                continue
+            keymap_dict[row[0]] = row[1]
```

Comment:
So, you assume that the mapping always be a keycode-to-keycode one, but in fact we do have keycode-to-keycode-and-scancode ones, and that should be considered as well.

Reply:
How about I change to keymap_dict[row[0]] = row[1:]?
 if for other cnv file, the key=key_name, value = [keycode, scancode]